pkg/auth/authproviders/provider_commands_test.go

```diff
@@ -71,8 +71,14 @@ func (s *mockStore) GetAuthProvider(_ context.Context, _ string) (*storage.AuthP
 	return s.expectedProvider, s.expectedFound, s.expectedErr
 }
 
-func (s *mockStore) GetAllAuthProviders(_ context.Context) ([]*storage.AuthProvider, error) {
-	return s.expectedProviders, s.expectedErr
+func (s *mockStore) ForEachAuthProvider(_ context.Context, fn func(obj *storage.AuthProvider) error) error {
```

Comment:
nit: in the current case, would it make sense to return early in case there is an expected error from the mock instead of processing all entries and eventually return the expected error ?

Reply:
It seems, though, that this mock function is not used.